.github/workflows/voiceover-test.yml

```diff
@@ -0,0 +1,143 @@
+name: voiceover-test
+on:
+  workflow_dispatch:
+    inputs:
+      aria_at_ref:
+        description: |
+          The sha / ref to checkout for aria-at tests
+        required: false
+        type: string
+      work_dir:
+        description: |
+          The --plan-workingdir passed to the aria-at-automation harness.
+          Based from the aria-at repo build folder.
+          Uses a default of tests/alert if not provided.
```

Comment:
Seems like failing on this being empty is the better alternative. Doesn't this risk providing completely unrelated results?

Reply:
The ability to quickly run a test run manually from GitHub interface is very useful (which is why the other workflow has the same default)

Reply:
@howard-e Thanks for voicing this concern. [I had the same thought when I reviewed the code upon which this is based](https://github.com/bocoup/aria-at-gh-actions-helper/pull/7#discussion_r1506785061), and I accepted "developer convenience" as sufficient justification. Reading your unease makes me want to revisit this question.

@gnarf [The GitHub UI prompts for input when triggering a workflow that has required input](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow):

> 6. If the workflow requires input, fill in the fields.

Would you agree this is sufficiently ergonomic for testing purposes?

Reply:
Assuming a good copy/paste default is in the description I guess this makes sense. we rarely do manual test starts, but having to fill in most of the fields will make starting one up the click/paste requirements of doing so.  I'm not supremely convinced either way so if two people are leaning "default could cause unwanted problems later" I'm inclined to agree

Reply:
I'd be happy with that solution. Thanks for linking the older discussion as well @jugglinmike

Seeing as it isn't affected just here, it being included in some follow up PR to update both workflow files seem just fine to me. Definitely not something to block this PR on.